src/Config.js

```diff
@@ -77,7 +80,23 @@ class Config {
     return mergedConfig;
   };
 
+  onConfigChange = (callback) => {
+    const watcher = chokidar.watch([DEFAULT_CONFIG_PATH, CONFIG_PATH]);
+    watcher.on('change', async () => {
+      let tempConfig = this._loadConfig();
+      // Check if onvif device config changed. Only this change observe
+      if (tempConfig?.onvif !== undefined && this._config?.onvif !== undefined) {
```

Comment:
Seeing as `onvif` is either an object or undefined, maybe we can shorten this to: 
```
      if (tempConfig?.onvif && this._config?.onvif) {
```